File: Programming Language(ver.2)/Go/308.go
package main
import "fmt"

func compute(n int) int {
	if n == 0{
		return 0
	}else if n == 1{
		return 1
	}else{
		return compute(n-1) + compute(n-2)
	}
}

func main(){
	var n int
	fmt.Scanln(&n)
	for i := n; i > 0; i--{
		fmt.Printf("fib(%d)=%d\n", i, compute(i))
	}
}

File: Programming Language(ver.2)/Python/308.py
fib = [0, 1]
def compute(x):
    global fib
    for i in range(x):
        fib.append(fib[-1] + fib[-2])
    return fib[x]

n = int(input())
Fib_n = compute(n)
for i in range(n, 0, -1):
    print('fib(' + str(i) + ')=' + str(fib[i]))